src/features/Account/LoginView.tsx

```diff
@@ -26,16 +26,16 @@ const LoginView = () => {
               fontSize: 'xl',
               fontWeight: 'bold',
               paddingTop: '4',
-              fontFamily: 'Poppins, sans-serif',
+              fontFamily: 'inherit',
```

Comment:
Is 'inherit' needed? Or would it just inherit as Poppins now that poppins is loaded into the app correctly?

Reply:
it's some weird behavior with the styles but inherit is usually needed to get the font from parent blocks